### ISSUE TITLE
Benefit Letters - Heading order violation - Alert heading  level 3 should be an h2

### ISSUE THREAD

**Issue by bacitracin** [department-of-veterans-affairs/va.gov-team#108006]
## Issue
"We don't have a valid address on file for you" alert H3 is invalid. It should be an H2

New Cypress Axe rules for headings are coming and will be put into effect April 30. [Slack thread](https://dsva.slack.com/archives/C03R5SBELQM/p1742920275472879) The link to the Axe report expired, but I noticed this one while running an Axe Devtools scan.

Page: https://staging.va.gov/records/download-va-letters/letters/letter-page

<img width="1491" alt="Image" src="https://github.com/user-attachments/assets/90fafa98-b72d-4ab6-948c-238c1d9ad5b9" />
